lighthouse-core/lib/i18n.js

```diff
@@ -58,6 +65,13 @@ function _preprocessMessageValues(icuMessage, values) {
 
   const clonedValues = JSON.parse(JSON.stringify(values));
   const parsed = MessageParser.parse(icuMessage);
+  // Throw an error if a message's value isn't provided
+  parsed.elements.map(el => el.id).filter(Boolean).forEach(valueId => {
```

Comment:
the type definition seems to imply all the elements have `id`. Is that not true?

Reply:
not sure, @paulirish did you find some without ids? :)

Reply:
@brendankenny correct the types are wrong, all things with a `format` have `id` but the strings in between don't, we'll switch to filter on `type === 'ArgumentElement'`

Reply:
can you also update the dfn in `typings/intl-messageformat-parser/index.d.ts`

Reply:
done